test/mlang/mlang.mc

```diff
@@ -87,8 +87,7 @@ lang FooCombined = FooA + FooTrans
 
 mexpr
 
-let _ =
-  use ArithBool2 in
+use ArithBool2 in
```

Comment:
I think this changes how far the `use` is valid, right? I think there is a point in keeping the same scoping here, to minimize risk of names in one language making a test for a later language work, just because the former happens to still be in scope, since these are tests that look at precisely that. I'm guessing parentheses around the `use` and its scope would be the easiest way.

The same should also apply to the other tests in `mlang`.

Reply:
Thanks. Fixed.